### DISCUSSION TITLE
move_hub.imu.angular_velocity()

### DISCUSSION THREAD

**Discussion by MorozovStVl** [pybricks/support#1043] [Q&A] [answered]
Hello!
I really like the Pybricks project. It expands our possibilities.

Tell me, please, for the block "Move Hub" is it planned to implement the function "imu.angular_velocity ()" or is it technically not possible?
I need it for the balancing robot.

**Comment by laurensvalk** [comment, marked as answer]
Unfortunately, the Move Hub does not have a gyroscopic sensor, so it cannot measure angular velocity.
